icons/sdl/sdl-plain.svg

```diff
@@ -0,0 +1 @@
25.18V36.659H79.4v29.064h15.705zM67.454 54.638c0 3.342-.767 6.338-2.305 8.982s-3.48 4.674-5.826 6.09a16.79 16.79 0 0 1-6.395 2.328c-2.305.347-4.758.519-7.36.519H33.766v-35.91h11.703c2.652 0 5.132.16 7.444.485a16.87 16.87 0 0 1 6.513 2.339 16.43 16.43 0 0 1 5.876 6.113c1.435 2.564 2.152 5.586 2.152 9.058zm-9.726-.072c0-2.366-.427-4.388-1.282-6.067s-2.163-2.995-3.938-3.945c-1.282-.675-2.629-1.042-4.041-1.107l-5.052-.099h-.179V65.83h.179l5.384-.111c1.458-.072 2.846-.504 4.167-1.29 1.553-.931 2.736-2.221 3.545-3.858s1.217-3.644 1.217-6.006zM15.687 73.201c-2.888 0-5.426-.267-7.612-.797s-4.216-1.19-6.094-1.976v-8.539h.862c1.893 1.53 3.976 2.701 6.258 3.522s4.491 1.232 6.635 1.232c.542 0 1.255-.057 2.141-.172s1.599-.29 2.141-.53a4.69 4.69 0 0 0 1.671-1.145c.458-.493.683-1.137.683-1.942 0-.9-.37-1.629-1.114-2.194s-1.629-.973-2.659-1.232l-4.117-.916a34.13 34.13 0 0 1-4.09-1.061c-2.923-.981-5.018-2.301-6.281-3.953s-1.9-3.732-1.9-6.223c0-3.362 1.492-6.082 4.483-8.165s6.719-3.125 11.195-3.125a34.76 34.76 0 0 1 6.7.652c2.22.435 4.209 1.023 5.968 1.763v8.2h-.836c-1.404-1.141-3.114-2.106-5.132-2.896s-4.09-1.183-6.219-1.183a13 13 0 0 0-2.228.172c-.656.115-1.351.328-2.08.649a4.19 4.19 0 0 0-1.53 1.149c-.431.508-.649 1.072-.649 1.698 0 .916.343 1.645 1.026 2.183s2.007 1.026 3.964 1.461l3.663.794c1.171.259 2.427.611 3.762 1.061 2.614.9 4.564 2.11 5.838 3.632s1.912 3.522 1.912 6.017c0 3.568-1.473 6.437-4.419 8.608S20.7 73.2 15.686 73.2zm0 0"/><path d="M31.163 89.112c-.233 0-.424.164-.424.366s.191.37.424.37.42-.168.42-.37-.187-.366-.42-.366zm18.101 3.293c-.362 0-.66.256-.66.572s.298.576.66.576.66-.256.66-.576-.294-.572-.66-.572zm-9.394-1.244c-.29 0-.527.206-.527.462s.237.458.527.458.53-.206.53-.458-.237-.462-.53-.462zm19.685 1.614c-.454 0-.824.321-.824.717s.37.717.824.717.824-.321.824-.717-.37-.717-.824-.717zm10.627-.553c-.572 0-1.034.401-1.034.897s.462.897 1.034.897 1.03-.404 1.03-.897-.462-.897-1.03-.897zm10.646-1.824c-.71 0-1.286.5-1.286 1.122s.576 1.118 1.286 1.118 1.29-.5 1.29-1.118-.576-1.122-1.29-1.122zm10.657-3.339c-.889 0-1.61.63-1.61 1.404s.721 1.4 1.61 1.4 1.614-.63 1.614-1.4-.725-1.404-1.614-1.404zm10.203-4.438c-.988 0-1.79.694-1.79 1.557s.801 1.557 1.79 1.557 1.79-.698 1.79-1.557-.801-1.557-1.79-1.557zm9.303-6.384c-1.099 0-1.992.775-1.992 1.728s.893 1.732 1.992 1.732 1.988-.775 1.988-1.732-.889-1.728-1.988-1.728zm7.669-8.982c-1.431 0-2.587 1.007-2.587 2.251s1.156 2.247 2.587 2.247 2.587-1.007 2.587-2.247-1.16-2.251-2.587-2.251zm3.293-9.318c-1.099 0-1.988.775-1.988 1.732s.889 1.728 1.988 1.728 1.992-.775 1.992-1.728-.889-1.732-1.992-1.732zm-1.11-5.815c.786 0 1.423-.553 1.423-1.232s-.637-1.236-1.423-1.236-1.419.553-1.419 1.236.637 1.232 1.419 1.232zm-3.468-6.719c.561 0 1.015-.393 1.015-.881s-.458-.881-1.015-.881-1.019.393-1.019.881.458.881 1.019.881zm-3.819-5.636c-.37 0-.675.267-.675.591s.305.588.675.588.679-.263.679-.588-.305-.591-.679-.591zm-4.06-2.308c.248 0 .45-.176.45-.389s-.202-.393-.45-.393-.45.176-.45.393.202.389.45.389" fill-rule="evenodd"/></svg>
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128" fill="#173354" xmlns:v="https://vecta.io/nano"><path d="M59.807 95.195c-14.252-.488-27.805-3.721-38.343-9.138-.992-.508-3.159-1.717-3.159-1.759 0-.016.344.145.763.362 5.158 2.64 11.642 4.896 18.189 6.327 10.382 2.274 21.753 2.881 32.895 1.748 10.516-1.065 20.341-3.655 28.728-7.566 7.544-3.521 13.58-8.024 17.602-13.118 6.173-7.834 7.002-16.914 2.274-24.969-3.06-5.22-8.169-9.822-15.182-13.667-.671-.366-1.199-.668-1.171-.668.092 0 3.06 1.431 3.987 1.927 4.529 2.404 8.131 4.968 11.275 8.017 3.949 3.831 6.429 7.776 7.643 12.16 1.121 4.052.912 8.547-.584 12.53-2.014 5.361-6.131 10.295-12.152 14.568-9.784 6.937-23.787 11.531-39.32 12.897-4.472.376-8.961.495-13.447.355zM95.441 76.83h-.076c-.034-.015-.084-.023-.153-.027l-.217-.008a1.26 1.26 0 0 0-.305.038l-.305.095v1.873h-.916V76h.916v.404a2.5 2.5 0 0 1 .176-.141 2.03 2.03 0 0 1 .202-.134 1.06 1.06 0 0 1 .229-.095c.092-.031.172-.042.252-.042h.099c.038.004.069.004.099.011zm-3.331 2.049c-.557 0-.981-.126-1.278-.382s-.443-.611-.443-1.08c0-.454.137-.82.412-1.091s.66-.412 1.156-.412c.45 0 .79.118 1.019.347s.34.561.34.992v.317h-2.011c.011.13.038.237.088.324a.56.56 0 0 0 .195.206c.076.053.172.092.275.114a1.65 1.65 0 0 0 .343.031 1.42 1.42 0 0 0 .313-.031 1.44 1.44 0 0 0 .282-.092c.076-.031.145-.065.21-.099l.156-.099h.095v.721l-.206.076a1.54 1.54 0 0 1-.256.076l-.305.057a2.81 2.81 0 0 1-.389.023zm.301-1.835c-.004-.187-.05-.324-.134-.424s-.214-.145-.389-.145-.317.053-.42.153-.153.24-.164.416zm-1.832-1.042l-1.484 3.827h-.984l.443-1.095-1.076-2.732h.962l.618 1.755.58-1.755zm-4.014 2.072v-.584l-.374.038c-.099.011-.191.031-.286.065-.08.027-.141.065-.187.118s-.065.126-.065.217c0 .13.038.221.111.271s.183.072.328.072a.66.66 0 0 0 .248-.053.73.73 0 0 0 .225-.145zm0 .431l-.168.137a1.01 1.01 0 0 1-.198.118c-.084.038-.164.069-.24.088a1.42 1.42 0 0 1-.328.031.9.9 0 0 1-.649-.252.82.82 0 0 1-.256-.626c0-.21.042-.378.13-.504s.214-.229.378-.305.362-.13.595-.16l.744-.069v-.015c0-.156-.061-.259-.179-.321s-.301-.088-.538-.088c-.111 0-.237.019-.382.057a2.76 2.76 0 0 0-.424.141h-.08v-.675a4.77 4.77 0 0 1 .443-.088 3.63 3.63 0 0 1 .611-.053c.507 0 .878.08 1.107.244s.347.404.347.733v1.904h-.912zm-1.709.298h-2.602V75.09h.981v3.007h1.622zm-5.174-.729v-.584l-.378.038a1.21 1.21 0 0 0-.282.065c-.08.027-.141.065-.187.118s-.069.126-.069.217c0 .13.038.221.114.271s.183.072.328.072a.64.64 0 0 0 .244-.053.8.8 0 0 0 .229-.145zm0 .431l-.168.137a1.05 1.05 0 0 1-.202.118 1.22 1.22 0 0 1-.24.088c-.076.019-.183.031-.324.031-.259 0-.477-.084-.649-.252a.83.83 0 0 1-.259-.626c0-.21.046-.378.134-.504s.214-.229.378-.305a2.05 2.05 0 0 1 .591-.16l.744-.069v-.015c0-.156-.057-.259-.179-.321s-.298-.088-.538-.088a1.52 1.52 0 0 0-.378.057 2.95 2.95 0 0 0-.427.141h-.076v-.675a4.69 4.69 0 0 1 .439-.088 3.6 3.6 0 0 1 .614-.053c.504 0 .874.08 1.107.244s.347.404.347.733v1.904h-.912zm-2.003.298h-.916v-2.801h.916zm.027-3.198h-.969v-.679h.969zm-1.32 3.198h-.916v-.29l-.202.149a1.32 1.32 0 0 1-.202.118c-.072.034-.145.057-.217.076s-.156.023-.256.023a.93.93 0 0 1-.797-.397c-.195-.263-.294-.614-.294-1.057 0-.244.034-.458.099-.637s.16-.336.275-.469a1.12 1.12 0 0 1 .389-.286c.153-.072.313-.107.481-.107a1.32 1.32 0 0 1 .382.046c.095.031.21.08.343.145v-1.194h.916zm-.916-.771v-1.377a1.03 1.03 0 0 0-.179-.053 1.29 1.29 0 0 0-.187-.019c-.221 0-.385.072-.496.217s-.168.351-.168.614c0 .275.046.477.134.599s.229.183.427.183a.8.8 0 0 0 .244-.042.99.99 0 0 0 .225-.122zm-3.114.847c-.553 0-.981-.126-1.274-.382s-.446-.611-.446-1.08c0-.454.141-.82.416-1.091s.66-.412 1.156-.412c.45 0 .79.118 1.019.347s.34.561.34.992v.317h-2.011c.011.13.038.237.088.324a.56.56 0 0 0 .195.206c.076.053.168.092.275.114s.217.031.343.031a1.42 1.42 0 0 0 .313-.031 1.44 1.44 0 0 0 .282-.092c.076-.031.145-.065.21-.099l.156-.099h.095v.721l-.206.076a1.67 1.67 0 0 1-.256.076l-.305.057a2.81 2.81 0 0 1-.389.023zm.305-1.835c-.008-.187-.05-.324-.134-.424s-.217-.145-.389-.145c-.179 0-.317.053-.42.153s-.156.24-.164.416zm-3.091 1.759v-1.397l-.008-.351c-.008-.095-.019-.172-.046-.233a.27.27 0 0 0-.122-.13c-.057-.027-.134-.042-.233-.042-.069 0-.137.015-.206.046a1.21 1.21 0 0 0-.225.122v1.984h-.92v-1.397l-.008-.347a.71.71 0 0 0-.05-.233c-.027-.057-.069-.107-.122-.134a.52.52 0 0 0-.229-.042c-.076 0-.149.019-.221.05l-.21.118v1.984h-.916V76h.916v.309c.153-.118.294-.214.427-.282a.94.94 0 0 1 .446-.103.92.92 0 0 1 .473.122.78.78 0 0 1 .313.366c.176-.156.343-.279.5-.362a1 1 0 0 1 .473-.126c.137 0 .259.023.366.065a.63.63 0 0 1 .275.187.85.85 0 0 1 .187.317 1.7 1.7 0 0 1 .061.488v1.82zm-4.296.057c-.374 0-.645-.076-.817-.221s-.252-.385-.252-.717v-1.309h-.351v-.611h.351v-.797h.916v.797h.881v.611h-.881v.992.256a.54.54 0 0 0 .042.198.3.3 0 0 0 .13.137c.061.034.153.053.271.053a.86.86 0 0 0 .195-.031.61.61 0 0 0 .164-.057h.08v.618l-.317.057c-.114.015-.248.023-.412.023zm-2.186.019a2.44 2.44 0 0 1-.668-.088c-.198-.057-.374-.149-.523-.267a1.21 1.21 0 0 1-.343-.458 1.55 1.55 0 0 1-.122-.649c0-.271.042-.496.134-.683a1.29 1.29 0 0 1 .359-.466 1.48 1.48 0 0 1 .519-.259 2.37 2.37 0 0 1 .618-.084c.164 0 .324.019.488.053s.324.095.488.172v.748h-.118a1.39 1.39 0 0 0-.134-.107c-.054-.038-.111-.076-.176-.111s-.141-.065-.225-.088a.91.91 0 0 0-.279-.038c-.233 0-.416.076-.542.233s-.195.366-.195.63c0 .279.069.488.206.626s.321.206.546.206a1.22 1.22 0 0 0 .305-.034c.088-.023.16-.057.225-.092s.111-.072.156-.107l.111-.103h.118v.748l-.179.076a1.91 1.91 0 0 1-.214.069l-.248.053a2.14 2.14 0 0 1-.309.019zm-2.797 0c-.557 0-.981-.126-1.278-.378s-.443-.614-.443-1.08c0-.458.137-.824.416-1.095s.66-.408 1.156-.408c.45 0 .79.114 1.015.347s.34.561.34.992v.313H59.49c.008.13.038.237.088.324a.56.56 0 0 0 .191.21c.08.05.172.088.275.111a1.6 1.6 0 0 0 .343.034 1.46 1.46 0 0 0 .317-.034c.103-.023.195-.053.282-.088l.206-.103.156-.095h.099v.717l-.206.08a2.03 2.03 0 0 1-.256.072c-.099.027-.202.046-.309.061a2.86 2.86 0 0 1-.385.019zm.305-1.835c-.004-.183-.053-.324-.137-.42s-.214-.145-.389-.145-.317.05-.416.153a.63.63 0 0 0-.168.412zm-1.843-.214h-.076c-.034-.011-.084-.019-.149-.027l-.221-.008a1.21 1.21 0 0 0-.301.038c-.107.027-.21.061-.309.095v1.877h-.916v-2.801h.916v.401a1.69 1.69 0 0 1 .176-.137 1.22 1.22 0 0 1 .202-.134c.065-.038.141-.072.233-.099a.86.86 0 0 1 .248-.042l.099.004a.71.71 0 0 1 .099.008zM56.4 78.806h-.916v-2.801h.916zm.027-3.201h-.969v-.679h.969zm-1.145 1.347a1.81 1.81 0 0 1-.24.931 1.74 1.74 0 0 1-.603.63c-.202.122-.42.202-.66.24a5.31 5.31 0 0 1-.759.053h-1.221v-3.713h1.21a5.36 5.36 0 0 1 .771.05 1.8 1.8 0 0 1 .672.24 1.69 1.69 0 0 1 .607.633c.145.263.225.576.225.935zm-1.007-.008c0-.244-.042-.454-.134-.626a.93.93 0 0 0-.404-.408 1 1 0 0 0-.42-.114l-.523-.011h-.019v2.324h.019l.557-.011a.89.89 0 0 0 .431-.134.94.94 0 0 0 .366-.397 1.39 1.39 0 0 0 .126-.622zm-5.136 1.935c-.553 0-.981-.126-1.274-.378s-.446-.611-.446-1.08c0-.458.141-.82.416-1.095s.66-.408 1.156-.408c.45 0 .79.114 1.015.347s.343.561.343.992v.313h-2.011c.008.13.038.24.088.328a.56.56 0 0 0 .195.206.75.75 0 0 0 .275.111 1.6 1.6 0 0 0 .343.034 1.43 1.43 0 0 0 .313-.034c.103-.023.198-.05.282-.088a1.64 1.64 0 0 0 .21-.103l.153-.095h.099v.721l-.206.076a1.67 1.67 0 0 1-.256.076l-.309.057c-.103.015-.233.019-.385.019zm.305-1.832c-.004-.187-.05-.328-.134-.424s-.217-.145-.389-.145c-.179 0-.317.05-.42.153s-.156.24-.164.416zm-2.19 1.759h-.916v-3.881h.916zm-1.084-1.438c0 .446-.122.809-.359 1.084s-.534.408-.881.408c-.145 0-.275-.015-.382-.046a1.56 1.56 0 0 1-.347-.149v1.168h-.916v-3.827h.916v.294a2.05 2.05 0 0 1 .412-.267 1.09 1.09 0 0 1 .485-.103c.34 0 .603.13.79.385s.282.611.282 1.053zm-.939.015c0-.271-.046-.469-.134-.591s-.225-.187-.412-.187a.9.9 0 0 0-.244.038c-.08.023-.16.065-.24.114v1.408c.05.019.103.034.16.042s.13.011.21.011c.221 0 .389-.069.496-.206s.164-.347.164-.63zm-3.228 1.423v-1.4l-.008-.347c-.004-.095-.019-.172-.046-.233s-.069-.103-.122-.13a.54.54 0 0 0-.233-.042.57.57 0 0 0-.206.042 1.48 1.48 0 0 0-.221.126v1.984h-.923v-1.4l-.008-.347a.73.73 0 0 0-.05-.233c-.027-.057-.065-.103-.122-.13a.52.52 0 0 0-.229-.042c-.072 0-.149.019-.217.05a1.54 1.54 0 0 0-.21.118v1.984h-.916v-2.801h.916v.309c.149-.122.29-.214.424-.282a.96.96 0 0 1 .446-.103c.179 0 .34.042.477.122s.237.202.309.362a2.66 2.66 0 0 1 .5-.359.99.99 0 0 1 .477-.126 1.02 1.02 0 0 1 .366.061.72.72 0 0 1 .275.187.95.95 0 0 1 .183.321c.042.126.061.286.061.485v1.824zm-3.892 0h-.916v-2.801h.916zm.027-3.198h-.969v-.683h.969zm-2.759 3.266c-.301 0-.561-.031-.79-.084s-.435-.122-.63-.202v-.885h.092c.195.16.408.279.645.366a2.05 2.05 0 0 0 .687.126c.057 0 .13-.008.221-.015a.88.88 0 0 0 .221-.057.46.46 0 0 0 .172-.118c.05-.05.072-.118.072-.198 0-.095-.038-.172-.114-.229a.8.8 0 0 0-.275-.126l-.427-.095c-.153-.031-.29-.069-.424-.111-.301-.099-.519-.237-.649-.408s-.195-.385-.195-.645c0-.347.153-.626.462-.843s.694-.321 1.156-.321a3.51 3.51 0 0 1 .694.065c.228.046.435.107.614.183v.847h-.084c-.145-.118-.321-.217-.53-.298a1.77 1.77 0 0 0-.645-.122 1.41 1.41 0 0 0-.229.015c-.069.011-.141.034-.214.069a.44.44 0 0 0-.16.118c-.046.05-.065.111-.065.176 0 .095.034.168.103.225s.21.107.412.153l.378.08a3.85 3.85 0 0 1 .389.111c.271.092.473.217.603.374s.198.366.198.622c0 .37-.153.668-.458.889s-.714.34-1.232.34zm59.727-6.31h-25.18V36.659H79.4v29.064h15.705zM67.454 54.638c0 3.342-.767 6.338-2.305 8.982s-3.48 4.674-5.826 6.09a16.79 16.79 0 0 1-6.395 2.328c-2.305.347-4.758.519-7.36.519H33.766v-35.91h11.703c2.652 0 5.132.16 7.444.485a16.87 16.87 0 0 1 6.513 2.339 16.43 16.43 0 0 1 5.876 6.113c1.435 2.564 2.152 5.586 2.152 9.058zm-9.726-.072c0-2.366-.427-4.388-1.282-6.067s-2.163-2.995-3.938-3.945c-1.282-.675-2.629-1.042-4.041-1.107l-5.052-.099h-.179V65.83h.179l5.384-.111c1.458-.072 2.846-.504 4.167-1.29 1.553-.931 2.736-2.221 3.545-3.858s1.217-3.644 1.217-6.006zM15.687 73.201c-2.888 0-5.426-.267-7.612-.797s-4.216-1.19-6.094-1.976v-8.539h.862c1.893 1.53 3.976 2.701 6.258 3.522s4.491 1.232 6.635 1.232c.542 0 1.255-.057 2.141-.172s1.599-.29 2.141-.53a4.69 4.69 0 0 0 1.671-1.145c.458-.493.683-1.137.683-1.942 0-.9-.37-1.629-1.114-2.194s-1.629-.973-2.659-1.232l-4.117-.916a34.13 34.13 0 0 1-4.09-1.061c-2.923-.981-5.018-2.301-6.281-3.953s-1.9-3.732-1.9-6.223c0-3.362 1.492-6.082 4.483-8.165s6.719-3.125 11.195-3.125a34.76 34.76 0 0 1 6.7.652c2.22.435 4.209 1.023 5.968 1.763v8.2h-.836c-1.404-1.141-3.114-2.106-5.132-2.896s-4.09-1.183-6.219-1.183a13 13 0 0 0-2.228.172c-.656.115-1.351.328-2.08.649a4.19 4.19 0 0 0-1.53 1.149c-.431.508-.649 1.072-.649 1.698 0 .916.343 1.645 1.026 2.183s2.007 1.026 3.964 1.461l3.663.794c1.171.259 2.427.611 3.762 1.061 2.614.9 4.564 2.11 5.838 3.632s1.912 3.522 1.912 6.017c0 3.568-1.473 6.437-4.419 8.608S20.7 73.2 15.686 73.2zm0 0"/><path d="M31.163 89.112c-.233 0-.424.164-.424.366s.191.37.424.37.42-.168.42-.37-.187-.366-.42-.366zm18.101 3.293c-.362 0-.66.256-.66.572s.298.576.66.576.66-.256.66-.576-.294-.572-.66-.572zm-9.394-1.244c-.29 0-.527.206-.527.462s.237.458.527.458.53-.206.53-.458-.237-.462-.53-.462zm19.685 1.614c-.454 0-.824.321-.824.717s.37.717.824.717.824-.321.824-.717-.37-.717-.824-.717zm10.627-.553c-.572 0-1.034.401-1.034.897s.462.897 1.034.897 1.03-.404 1.03-.897-.462-.897-1.03-.897zm10.646-1.824c-.71 0-1.286.5-1.286 1.122s.576 1.118 1.286 1.118 1.29-.5 1.29-1.118-.576-1.122-1.29-1.122zm10.657-3.339c-.889 0-1.61.63-1.61 1.404s.721 1.4 1.61 1.4 1.614-.63 1.614-1.4-.725-1.404-1.614-1.404zm10.203-4.438c-.988 0-1.79.694-1.79 1.557s.801 1.557 1.79 1.557 1.79-.698 1.79-1.557-.801-1.557-1.79-1.557zm9.303-6.384c-1.099 0-1.992.775-1.992 1.728s.893 1.732 1.992 1.732 1.988-.775 1.988-1.732-.889-1.728-1.988-1.728zm7.669-8.982c-1.431 0-2.587 1.007-2.587 2.251s1.156 2.247 2.587 2.247 2.587-1.007 2.587-2.247-1.16-2.251-2.587-2.251zm3.293-9.318c-1.099 0-1.988.775-1.988 1.732s.889 1.728 1.988 1.728 1.992-.775 1.992-1.728-.889-1.732-1.992-1.732zm-1.11-5.815c.786 0 1.423-.553 1.423-1.232s-.637-1.236-1.423-1.236-1.419.553-1.419 1.236.637 1.232 1.419 1.232zm-3.468-6.719c.561 0 1.015-.393 1.015-.881s-.458-.881-1.015-.881-1.019.393-1.019.881.458.881 1.019.881zm-3.819-5.636c-.37 0-.675.267-.675.591s.305.588.675.588.679-.263.679-.588-.305-.591-.679-.591zm-4.06-2.308c.248 0 .45-.176.45-.389s-.202-.393-.45-.393-.45.176-.45.393.202.389.45.389" fill-rule="evenodd"/></svg>

```